api/ServiceWorkerContainer.json

```
@@ -410,6 +410,44 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "enforces_trusted_types": {
+          "__compat": {
+            "description": "`scriptURL` parameter requires `TrustedScriptURL` instance when trusted types are enforced",
+            "support": {
+              "chrome": {
+                "version_added": "140"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "136",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.security.trusted_types.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              "firefox_android": "mirror",
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "26"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
```

`ServiceWorkerContainer.register()` is not supported in WebView iOS, hence the failing test:

```suggestion
              "webview_ios": {
                "version_added": false
              }
```